### PR TITLE
[doc] add tgi_compat and see details

### PR DESCRIPTION
**Pull request by sindhuvahinis**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: serving/docs/lmi/user_guides/lmi_input_output_schema.md [hunk 1]
```diff
@@ -122,6 +122,40 @@ When using streaming:
...
+            "finish reason": "length",
+            "generated_tokens": 8,
+            "inputs": "What is Deep Learning?",
+            "tokens": [<Token1>, <Token2>, ...]
+        }
+    }
+]
+```
+
+When using streaming: Response will be Server Sent Events (text/event-stream) which will prefix with `data:`
```
Comment (lanking520) [inline serving/docs/lmi/user_guides/lmi_input_output_schema.md:148]: this is the error case?

Comment (sindhuvahinis) [reply serving/docs/lmi/user_guides/lmi_input_output_schema.md:148]: No, this is a separate heading 2.

Comment (lanking520) [reply serving/docs/lmi/user_guides/lmi_input_output_schema.md:148]: Understand, but the below sample is a demo of error condition?

Comment (sindhuvahinis) [reply serving/docs/lmi/user_guides/lmi_input_output_schema.md:148]: Lol, Changed it

**Review by lanking520 — APPROVED** [review]

**Pull request merged**